src/main/kotlin/org/rust/cargo/service/CargoInstallationManager.kt

```diff
@@ -1,29 +0,0 @@
-package org.rust.cargo.service
-
-import com.intellij.openapi.projectRoots.SdkType
-import org.rust.cargo.util.PlatformUtil
-import org.rust.cargo.project.RustSdkType
-
-class CargoInstallationManager {
-
-    fun hasCargoMetadata(cargoHomePath: String?): Boolean {
```

Comment:
Hm, looks like it wasn't exactly dead, but the return code for missing subcommand changed from `127` to `101`.